backend/src/mockExamAttempt/entities/mockExamAttempt.entity.ts

```diff
@@ -38,4 +39,10 @@ export class MockExamAttempt {
 
   @Column()
   isPassed: boolean;
+
+  @ManyToOne(() => DriversLicenseCategory)
```

Comment:
I assume this can't be optional, right? If so let's set `nullable: true`.
In the migration you'll probably need to set the driver's license category. Please take a look here, how it could be done: https://github.com/twoje-osk/twoje-osk/blob/maciek/feature/mock-exams-categories/backend/src/migrations/1673803541669-TraineesRelationWithLicenses.ts